README.md

```diff
@@ -1 +1,10 @@
-# twitter-clone-mariam
\ No newline at end of file
+
+# twitter-clone-mariam
+ oreo website  :A social networking site that allows a user to have short posts of up to 300 characters called tweets. It also allows the user to send messages between his friends. The user can follow other people and see their tweets, like them and share them.
+ # website  sketches
+ ![sketches](https://user-images.githubusercontent.com/52622730/98227997-ade94700-1f60-11eb-9f1c-30c9a4cf045e.jpg)
```

Comment:
This is a sketch for twitter not for your clone!!! 
edit it to be your copy, not the original

Reply:
In the document, we describe what features you should add, read them again to edit your sketch